stable/spark/docker_images/livy/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# FROM ubuntu:16.04
+FROM java:openjdk-8-jdk
```

Comment:
My other thought here is… is the chart the best place to store these Dockerfiles and their scripts? Keeping in VCS somewhere separate means gitops work can be done on the images without needing to bump the chart version on every dev change (testing could be automated, etc).

Reply:
Thanks for the feedback. I think you have a good point here. We will move the dockerfiles to another repo and put a link to the sources